test/onnx/test_operators.py

```diff
@@ -352,6 +355,46 @@ def test_clip(self):
         x = torch.randn(3, 4, requires_grad=True)
         self.assertONNX(lambda x: torch.clamp(x, min=-0.5, max=0.5), x)
 
+    @skipIfCaffe2
+    def test_clip_aten_fallback_due_exception(self):
+        _onnx_opset_version = 9
+        x = torch.randn(3, 4, requires_grad=True)
+
+        def bad_clamp(g, self, min, max):
+            return _onnx_unsupported("Bad boy!")
+
+        register_custom_op_symbolic('::clamp', bad_clamp, _onnx_opset_version)
+        try:
+            self.assertONNX(lambda x: torch.clamp(x, min=-0.5, max=0.5),
+                            x,
+                            operator_export_type=torch.onnx.OperatorExportTypes.ONNX_ATEN_FALLBACK,
+                            opset_version=_onnx_opset_version)
+        finally:
+            unregister_custom_op_symbolic('::clamp', _onnx_opset_version)
+
+    @skipIfCaffe2
+    def test_clip_aten_fallback_explicit_request(self):
+        from torch.onnx import symbolic_registry
+
+        def is_registered_op_tampered(opname, domain, version):
+            fake_missing_symbolics = ('clamp')
```

Comment:
This is not creating a tuple, I'm pretty sure this is a bug.
You need a trailing comma `("clamp",)`